src/args/agnostic.nix

```diff
@@ -10,7 +10,13 @@
   fix' = __unfix__: let x = __unfix__ x // {inherit __unfix__;}; in x;
   sources = import ../nix/sources.nix;
   args = fix' (self: let
-    __nixpkgs__ = import sources.nixpkgs {inherit system;};
+    __nixpkgs__ = import sources.nixpkgs {
+      inherit system;
+      config = {
+        allowUnfree = true;
+        permittedInsecurePackages = ["ruby-2.7.8" "openssl-1.1.1w"];
```

Comment:
Maybe drop ruby 2.7 support?

What is requiring openssl-1.1.1?

Reply:
We can drop the support because we are adding now ruby 3.1, 3.2 and 3.3

Reply:
Lets only keep officially supported versions https://www.ruby-lang.org/en/downloads/branches/

Reply:
Hopefully we can also get rid of that one if we deprecate 2.7

Reply:
gotcha!

Reply:
removed!